content/en/docs/instrumentation/js/getting-started/browser.md

```diff
@@ -61,17 +61,17 @@ tsc --init
 ```
 Create an empty code file named `document-load` with a `.ts` or `.js` extension, as appropriate, based on the language you've chosen to write your app in. Add the following code to your HTML right before the `</body>` closing tag:
 
-{{< ot-tabs TypeScript JavaScript >}}
+{{< tabpane >}}
```

Comment:
When the language is the same in both tabs, you can factor that out (note that again here, the quotes are optional around the value "html"):

```suggestion
{{< tabpane lang=html >}}
```